Test/comp/Arrays.dfy.go.expect

```diff
@@ -0,0 +1,96 @@
+0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 
+17
+[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]
+[2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
```

Comment:
Why is Go so different starting from here?!

Reply:
The only difference is due to how Go prints strings when `--unicode-char` is `false`:

```
% diff Test/comp/Arrays.dfy.expect Test/comp/Arrays.dfy.go.expect 
88c88
< DDDDDDDDDDagggggaggg
---
> [D, D, D, D, D, D, D, D, D, D, a, g, g, g, g, g, a, g, g, g]
```

That does mean the NONUNIFORM tag on `Arrays.dfy` itself is pointing to the wrong issue though. I am second-guessing the decision to have the conversion script point to https://github.com/dafny-lang/dafny/issues/4108 by default, I think it would be better to force a manual review of any inconsistent output.